src/components/Text.js

```diff
@@ -0,0 +1,13 @@
+import styled, { css } from "styled-components";
+import theme from "../theme";
+
+export const Text = styled("h1").attrs(({ level }) => ({
```

Comment:
You can use it like this for brevity:

```suggestion
export const Text = styled.h1.attrs(({ level }) => ({
```